lambdatimes/src/components/Content/Tab.js

```diff
@@ -6,10 +6,12 @@ const Tab = props => {
       if it is not it should just be 'tab'*/
   return (
```

Comment:
Make sure that the props that you're passing into this Tab component match the props you're giving it in the Tabs component